src/config.js

```diff
@@ -1,3 +1,13 @@
+#-------------------------------------------------------------------------------
```

Comment:
Javascript uses `//` for single line comments `/*` for multilines, so this will throw errors. 

This should be 
```
/*
 multi
 line 
 comment
*/
```